e2e/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
```

Comment:
IMO we can remove some of these because they are not relevant to this folder.

Reply:
You were right, a lot was copypasta 🍝 we slurped it all up and left the garnish